python/kserve/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "kserve"
-version = "0.12.0"
+version = "0.13.0"
```

Comment:
ah, good catch - have been updating this locally so the package that depends on my updates to kserve pulls my version rather than the latest (0.12.0) from pypi.  will revert.